### ISSUE TITLE
Ente Auth Windows Issue !

### ISSUE THREAD

**Issue by discoveringaryan** [ente-io/ente#2879]
### Description

When opening Ente Auth on Windows 11 23H2 it prevents from opening/installing as it doesnt recognise the publisher, app, etc. There is no option to even "Run Anyway" . Maybe try bringing this app on microsoft store just like you did on Google Play Store ASAP !

But also do something about the standalone/portable windows app

### What product are you using?

Ente Auth

### What platform are you using?

Desktop - Windows

**Comment by prateekmedia** [comment]
Yeah the windows scanner gives false negatives a lot. You can install the app from chocolatey if you want.

**Comment by vishnukvmd** [comment]
Duplicate of #2496
